### PR TITLE
Basic Scalafiddle integration

### PR DESCRIPTION
**Pull request by andreaTP**
Added a basic integration with ScalaFiddle.

### CODE REVIEW

**Comment by andreaTP** [comment]
targeting #12 :-)
a bit of refactoring (probably need more at some point)

File: core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala [hunk 1]
```diff
@@ -1,12 +1,12 @@
 /*
  * Copyright © 2015 - 2016 Lightbend, Inc. <http://www.lightbend.com>
- * 
+ *
```
Comment (eed3si9n) [inline core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala]: Was this whitespace manually edited? It should be managed by sbt-header.

**Comment by eed3si9n** [comment]
Could you add tests please?

**Comment by andreaTP** [comment]
fixed space in header and added some basic test...
please let me know if you need more or a minimum coverage or whatever, I'm not confident that everything is stable and things will not change soon, so I start slow ... 

b.t.w.
great tool! thanks!

File: core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala [hunk 2]
```diff
@@ -135,6 +135,58 @@ object SnipDirective {
...
+           | }
+          """.stripMargin, "UTF-8")
+
+      printer.println.print(s"""
+        <iframe class="$cssClass" $width $height src="$baseUrl?$extraParams&source=$fiddleSource" frameborder="0" style="$cssStyle"></iframe>
+        """
+      )
+    } catch {
+      case e: FileNotFoundException =>
+        throw new SnipDirective.LinkException(s"Unknown fiddle [${e.getMessage}] referenced from [${page.path}]")
```
Comment (pvlugter) [inline core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala]: Looks like this should be `FiddleDirective.LinkException`.

**Comment by pvlugter** [comment]
Cool. Nice feature.

File: core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala (hunk 2 above)
Comment (andreaTP) [inline core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala]: true! thanks

**Comment by andreaTP** [comment]
fix and rebased

**Comment by andreaTP** [comment]
the failed test isn't related to this feature:
```
[info] - should parse and render all header children *** FAILED ***
[info]   "...h
[info]   <code>code</code>&[amp;]hellip;</h1>
[info]   
[info]   " did not equal "...h
[info]   <code>code</code>&[]hellip;</h1>
[info]   
[info]   " (AnchorLinkSpec.scala:29)
```

**Comment by andreaTP** [comment]
Anything yet I should do here?

**Comment by eed3si9n** [comment]
Restarted the Travis job.

**Comment by eed3si9n** [comment]
Merging because testing locally succeeded, but I'm guessing there's some race condition on the tests that we need to take care of eventually.

**Comment by pvlugter** [comment]
Haven't seen test failures like this before either, locally or in CI. In the past it was running on Lightbend/Typesafe Jenkins builds, not Travis. Looks like it's always one of the http render checks, but not the same one?

**Comment by andreaTP** [comment]
thanks guys!

**Pull request merged**